### PR TITLE
Feat/rolluxtestnet

### PR DESCRIPTION
**Pull request by ThimSouza**

### CODE REVIEW

**Review by claudiovb — CHANGES_REQUESTED** [review]
Partial Review 1

**Review by ThimSouza — COMMENTED** [review]
Reviewed with Claudio

**Review by ThimSouza — COMMENTED** [review]
removed unused files and folders from github

File: package-lock.json
Comment (Potolski) [inline package-lock.json:1]: Remove this file. We already have yarn.lock and having both will most likely cause problems

**Pull request merged**